src/index.spec.ts

```diff
@@ -1,12 +1,227 @@
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+enableFetchMocks();
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
 test('Testing time!', () => {
   console.log('welcome');
 });
+
+describe('isInteger test', () => {
```

Comment:
You could have added a test for strings, null, and NaN values. 

toBeTruthy will match with any truthy value, including an empty array or object. To be more specific you could have used `toBeTrue()` (from jest-extended) or `toBe(true)` (without jest-extended).